test/katex-spec.js

```diff
@@ -1354,7 +1354,18 @@ describe("An array environment", function() {
     it("should accept a single alignment character", function() {
         var parse = getParsed("\\begin{array}r1\\\\20\\end{array}");
         expect(parse[0].type).toBe("array");
-        expect(parse[0].value.cols).toEqual([{align:"r"}]);
+        expect(parse[0].value.cols).toEqual([
+            { type: "align", align: "r" }
+        ]);
+    });
+
+    it("should accept vertical separators", function() {
+        var parse = getParsed("\\begin{array}{c|}\\end{array}");
```

Comment:
It would be good to have tests that verify that it handles separators at initial position as well as multiple separators in a row.  I know that the screenshot test cover those cases, but the screenshot tests don't get run as often.